### ISSUE TITLE
Links do not have a discernible name to ensure accessibility

### ISSUE THREAD

**Issue by max-hopf** [MaxHopf/fusspflegezsuzsi-landingpage#3]
### Problem
Missing discernible name in anchor tag in line 174 of the index.html to ensure seamless experience for screen reader user. 

![image](https://github.com/user-attachments/assets/2641e119-dd4b-496c-a74b-aa4b435982ab)

### Solution
[Checkout how to make links accessible](https://dequeuniversity.com/rules/axe/4.9/link-name)
